### PR TITLE
Fixes for minor whitespace and scan tools

### PR DESCRIPTION
**Pull request by dgarske**
* Fixes breaks from PR #4807

### CODE REVIEW

File: src/ssl.c [hunk 1]
```diff
@@ -24091,10 +24094,10 @@ int wolfSSL_DupSession(const WOLFSSL_SESSION* input, WOLFSSL_SESSION* output,
             }
             else {
                 tmp = (byte*)XREALLOC(ticBuff, input->ticketLen,
-                        output->heap, DYNAMIC_TYPE_SESSION_TICK);
+                        input->heap, DYNAMIC_TYPE_SESSION_TICK);
```
Comment (SparkiDev) [inline src/ssl.c:24097]: ticBuff comes from output and is put back into output.
Use output->heap.

**Review by SparkiDev — CHANGES_REQUESTED** [review]

File: src/internal.c [hunk 2]
```diff
@@ -30626,7 +30626,7 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
                 id = ssl->session->altSessionID;
                 idSz = ID_LEN;
             }
-            XMEMCPY(it.id, id, ID_LEN);
+            XMEMCPY(it.id, id, idSz);
```
Comment (julek-wolfssl) [inline src/internal.c:30629]: I feel there should be a sanity check that `idSz < ID_LEN` for this change.

File: src/internal.c [hunk 3]
```diff
@@ -30765,28 +30765,28 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
         /* get master secret */
         if (ret == WOLFSSL_TICKET_RET_OK || ret == WOLFSSL_TICKET_RET_CREATE) {
             if (ssl->version.minor < it->pv.minor) {
-                ForceZero(&it, sizeof(it));
+                ForceZero(it, sizeof(*it));
```
Comment (julek-wolfssl) [inline src/internal.c:30768]: Interesting that I missed this. Yeah this is an important change to make sure we clear the data not the pointer.

File: src/ssl.c [hunk 4]
```diff
@@ -15763,8 +15768,6 @@ int wolfSSL_SetSession(WOLFSSL* ssl, WOLFSSL_SESSION* session)
         SESSION_ROW_UNLOCK(sessRow);
         sessRow = NULL;
     }
-    /* Make sure we don't access this anymore */
-    session = NULL;
```
Comment (julek-wolfssl) [inline src/ssl.c:15767]: I added this to make sure we don't slip up in the future and try to access this var after this point. Maybe at least leave a nice big comment that `session` can't be accessed after this point?

**Review by julek-wolfssl — CHANGES_REQUESTED** [review]
Thanks for fixing my mistakes! Just a few comments.

File: src/ssl.c (hunk 4 above)
Comment (dgarske) [reply src/ssl.c:15767]: Sounds good! I'll put a big comment instead.

File: src/internal.c (hunk 2 above)
Comment (julek-wolfssl) [reply src/internal.c:30629]: Yes, that `memset` means we don't need to fill the remaining space.

**Review by SparkiDev — APPROVED** [review]

**Review by julek-wolfssl — APPROVED** [review]

**Pull request merged**